lib/disability_compensation/requests/form526_request_body.rb

```diff
@@ -0,0 +1,305 @@
+# frozen_string_literal: true
```

Comment:
Just note here that this file is especially long because it is a conversion from a very long Lighthouse submission request. It's very simple, with no business logic or anything- just structure. We also built a test file for this, but it was also very long and would have (far) exceeded the 500 line limit. This will be added in a separate PR after this PR has been merged in.

Reply:
Understood, but in the future, personally I'd rather see the specs in the same file and override LOC check. IMO it's better to have a spec alongside with the code it's testing. I think most approvers would agree.

Reply:
Thanks, good to know for next time. We didn't think that the LOC check was overridable. 2nd PR coming soon.